Examples/Algorithms/TrackFinding/src/TrackParamsEstimationAlgorithm.cpp

```diff
@@ -76,6 +76,13 @@ ActsExamples::TrackParamsEstimationAlgorithm::createSeeds(
     const ActsExamples::SimSpacePointContainer& spacePoints) const {
   SimSeedContainer seeds;
   seeds.reserve(protoTracks.size());
+
+  std::map<Index, const SimSpacePoint*> spMap;
```

Comment:
Would `std::unordered_map` work/make sense here as well?

Reply:
Good idea, that would be even better. One moment!

Reply:
New version pushed with `std::unordered_map`, which indeed a little faster in this case. :smile: